inst/schema/RunRequest.schema.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I believe that at present we allow multiple changelog entries to be passed to orderly_run via message; we might expand this to be an array of changelog entries?

Reply:
After discussing, orderly CLI not supporting multiple changelog messages at the moment so let's continue with 1 for now. If science ask for multiple we can add that as an enhancement in the future. https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-4600